project/MiMa.scala

```diff
@@ -14,7 +14,7 @@ object MiMa extends AutoPlugin {
 
   //  akka-pki artifact was added in Akka 2.6.6
   private val firstPatchOf26 = 6
-  private val latestPatchOf26 = 20
+  private val latestPatchOf26 = 21
```

Comment:
Do we know why we had to change the mima files to 2.7.0?

I know there is some (difficult to grasp) logic for minor versions and filters must be added to several of them.
In this case, did you try to rename the files to 2.6.21?

Reply:
I didn't, I just expected that those binary incompatible changes must have happened in 2.7.0 and thought it's better to exclude after that in case we'd do another 2.6 release than to keep having to move them along with patch releases.

Reply:
Ok, sounds good. If the changes were in 2.7.0 we could not have added the filters in 2.7.0, but now we can. All good.